common/src/registers/MachRegister.C

```diff
@@ -1117,144 +1225,204 @@ namespace Dyninst {
+          case 70: return Dyninst::x86_64::xmm19;
+          case 71: return Dyninst::x86_64::xmm20;
+          case 72: return Dyninst::x86_64::xmm21;
+          case 73: return Dyninst::x86_64::xmm22;
+          case 74: return Dyninst::x86_64::xmm23;
+          case 75: return Dyninst::x86_64::xmm24;
+          case 76: return Dyninst::x86_64::xmm25;
+          case 77: return Dyninst::x86_64::xmm26;
+          case 78: return Dyninst::x86_64::xmm27;
+          case 79: return Dyninst::x86_64::xmm28;
+          case 80: return Dyninst::x86_64::xmm29;
+          case 81: return Dyninst::x86_64::xmm30;
+          case 82: return Dyninst::x86_64::xmm31;
+          case 83: return Dyninst::InvalidReg;
+          case 84: return Dyninst::InvalidReg;
+          case 85: return Dyninst::InvalidReg;
+          case 86: return Dyninst::InvalidReg;
+          case 87: return Dyninst::InvalidReg;
+          case 88: return Dyninst::InvalidReg;
+          case 89: return Dyninst::InvalidReg;
+          case 90: return Dyninst::InvalidReg;
+          case 91: return Dyninst::InvalidReg;
+          case 92: return Dyninst::InvalidReg;
+          case 93: return Dyninst::InvalidReg;
+          case 94: return Dyninst::InvalidReg;
+          case 95: return Dyninst::InvalidReg;
+          case 96: return Dyninst::InvalidReg;
+          case 97: return Dyninst::InvalidReg;
+          case 98: return Dyninst::InvalidReg;
+          case 99: return Dyninst::InvalidReg;
+          case 100: return Dyninst::InvalidReg;
+          case 101: return Dyninst::InvalidReg;
+          case 102: return Dyninst::InvalidReg;
+          case 103: return Dyninst::InvalidReg;
+          case 104: return Dyninst::InvalidReg;
+          case 105: return Dyninst::InvalidReg;
+          case 106: return Dyninst::InvalidReg;
+          case 107: return Dyninst::InvalidReg;
+          case 108: return Dyninst::InvalidReg;
+          case 109: return Dyninst::InvalidReg;
+          case 110: return Dyninst::InvalidReg;
+          case 111: return Dyninst::InvalidReg;
+          case 112: return Dyninst::InvalidReg;
+          case 113: return Dyninst::InvalidReg;
+          case 114: return Dyninst::InvalidReg;
+          case 115: return Dyninst::InvalidReg;
+          case 116: return Dyninst::InvalidReg;
+          case 117: return Dyninst::InvalidReg;
+          case 118: return Dyninst::x86_64::k0;
+          case 119: return Dyninst::x86_64::k1;
+          case 120: return Dyninst::x86_64::k2;
+          case 121: return Dyninst::x86_64::k3;
+          case 122: return Dyninst::x86_64::k4;
+          case 123: return Dyninst::x86_64::k5;
+          case 124: return Dyninst::x86_64::k6;
+          case 125: return Dyninst::x86_64::k7;
+          case 126: return Dyninst::InvalidReg;
+          case 127: return Dyninst::InvalidReg;
+          case 128: return Dyninst::InvalidReg;
+          case 129: return Dyninst::InvalidReg;
+
```

Comment:
The Oct 2023 ABI doc maps 130-145 to r16-r31 (the new registered defined by APX). These can likely be ignored for now.

Reply:
Yup. Capstone doesn't support AMX yet because it's based on LLVM 15 which has very poor support for them. The next release of Capstone will be based on at least LLVM 17 which has full support. I'll probably add the TMM registers soon just to have them.